docs/CONTRACT.md

```diff
@@ -109,6 +116,14 @@ What's most important here is that users have to opt-in to the migration process
 
 Note that the function `acceptMigration()` may also be used to stake on behalf of a different user - one user calls the function and another user is accredited for the stake. The function is named `acceptMigration()` and not `stakeOnBehalf()` to prevent phishing attempts where a user, unintendedly, stakes to another account instead of her own.
 
+&nbsp;
+### Stake Change Notifier
+The staking contract implements a notifier functionality used to notify stake changes and migrations to Orbs election contracts. Any change in an staking owner stake as a result of `stake`, `unstake`, `withdraw`, `withdrawReleasedStakes()`, `distributeRewards()`, `acceptMigration` is notified. The notification includes the stake owner address, stake change and the new stake amount. The migration of stake by the owner to a new contract is also notified.
```

Comment:
You have `()` in some functions (e.g., `distributeRewards()`), but not in others (e.g., `stake`).